packages/runtime/lib/app.js

```diff
@@ -113,6 +113,7 @@ class PlatformaticApp {
       await this.server.close()
     }
 
+    await this.config.configManager.stopWatching()
```

Comment:
This should probably happen in `stopFileWatching()` on the next line. That function currently calls `this.server.platformatic.configManager.stopWatching()`, which is actually the same object. That call is in an `if` statement now though, so maybe it shouldn't be.